optimade/server/routers/landing.py

```diff
@@ -1,39 +1,88 @@
-""" OPTIMADE landing page, rendered as a Jinja2 template. """
+""" OPTIMADE landing page router. """
 
 from pathlib import Path
+from functools import lru_cache
 
 from fastapi import Request
-from fastapi.templating import Jinja2Templates
+from fastapi.responses import HTMLResponse
 from starlette.routing import Router, Route
 from optimade import __api_version__
 
 from optimade.server.routers import ENTRY_COLLECTIONS
 from optimade.server.routers.utils import meta_values, get_base_url
 from optimade.server.config import CONFIG
 
-template_dir = Path(__file__).parent.joinpath("static").resolve()
-TEMPLATES = Jinja2Templates(directory=[template_dir])
 
+@lru_cache()
+def render_landing_page(url: str) -> HTMLResponse:
+    """Render and cache the landing page.
 
-async def landing(request: Request):
-    """Show a human-readable landing page when the base URL is accessed."""
+    This function uses the template file `./static/landing_page.html`, adapted
+    from the original Jinja template. Instead of Jinja, some basic string
+    replacement is used to fill out the fields from the server configuration.
 
-    meta = meta_values(request.url, 1, 1, more_data_available=False)
+    !! warning
+    The removal of Jinja means that the fields are no longer validated as
+    web safe before inclusion in the template.
```

Comment:
If this should be a properly formatted "warning" box in the documentation it should instead be something like:
```suggestion
    Warning:
        The removal of Jinja means that the fields are no longer validated as
        web safe before inclusion in the template.
```

Reply:
Looking elsewhere in the code I think we do still use the `!!!` admonitions, but my one was still wrong:

```suggestion
    !!! warning "Careful"
        The removal of Jinja means that the fields are no longer validated as
        web safe before inclusion in the template.
```

Reply:
Whatever you wish :) But indeed better to use the syntax we have used throughout the code base 👍